config.py

```diff
@@ -1,4 +1,4 @@
-listen_port = 8214
+listen_port = 8212
```

Comment:
It's only included to keep the default values in line with the example you've given in the readme, I feel like it better clarifies what the variable does (You can remove it if you want :) )